content/en/apps/tutorials/local-setup.md

```diff
@@ -94,12 +97,23 @@ This CHT instance is empty and has no data in it. While you're free to explore a
 
 ### 2. Install cht-conf
 
-Using npm and python on your terminal, install cht-conf and pyxform globally using the following commands:
-
+Using npm on your terminal, install cht-conf globally using the command below. Confirm if you have previously installed cht-conf.
+If you have, skip this step.
```

Comment:
I would say "If you have previously installed cht-conf, skip this step."

Reply:
I would remove this addition.  This whole guide is has many things that need to be installed and it'll end up being quite laborious to say "Install docker (skip if you've installed docker)" and "Install nodejs (skip if you've installed nodejs)". 

These steps are both harmless to run a again and are not time consuming for a beginner to re-run if they're unsure.

Brevity is the soul of wit! ;)